src/transaction/transaction.ts

```diff
@@ -298,15 +298,15 @@ export async function populateAppCallResources(atc: algosdk.AtomicTransactionCom
 
     // Do accounts first because the account limit is 4
     r.accounts?.forEach((a) => {
-      group[i].txn.appAccounts?.push(algosdk.decodeAddress(a))
+      group[i].txn.appAccounts = [...(group[i].txn.appAccounts ?? []), algosdk.decodeAddress(a)]
```

Comment:
```suggestion
      group[i].txn.appAccounts = (group[i].txn.appAccounts ?? []).concat(algosdk.decodeAddress(a))
```
Alternatively you can use concat.

Reply:
Ah forgot about concat. Some quick google-fu tells me there isn't a significant tradeoff here so just keeping as is.

Reply:
@joe-p Yeah nothing massive in this context. There is when you use the spread operator with `add` for example https://jsfiddle.net/y0fjxLv5/.